tests/tests_integration/test_api/test_geospatial.py

```diff
@@ -0,0 +1,430 @@
+import random
+import sys
+import time
+import uuid
+
+import pytest
+from cognite.client import utils
+from cognite.client.exceptions import CogniteAPIError
+
+from cognite.client.data_classes.geospatial import (
+    CoordinateReferenceSystem,
+    Feature,
+    FeatureList,
+    FeatureType,
+    FeatureTypeUpdate,
+    OrderSpec,
+    PropertyAndSearchSpec,
+)
+
+# sdk integration tests run concurrently on 3 python versions so this makes the CI builds independent from each other
+FIXED_SRID = 121111 + sys.version_info.minor
```

Comment:
This is fine for now, but we will run into race conditions since people run integration tests against this project locally and there may be multiple PRs with concurrent CI builds. So ideally every run should be completely independent.

Reply:
Maybe just add a TODO

Reply:
I don't know how to make it completely independent (any suggestion?), the space for SRIDs is limited, and a number of them is preallocated, but I'll add the TODO.